### PR TITLE
Fix characteristic tag output.

### PR DESCRIPTION
**Pull request by scovetta**
Fix the characteristic tag output to show just the keys, and not the key + value.

### CODE REVIEW

File: src/oss-characteristics/CharacteristicTool.cs [hunk 1]
```diff
@@ -193,7 +193,7 @@ private static List<string> GetTextResults(PackageURL purl, Dictionary<string, A
                     stringOutput.Add(string.Format("Programming Language: {0}",
                         string.Join(", ", metadata?.Languages?.Keys ?? Array.Empty<string>().ToList())));
                     stringOutput.Add("Unique Tags: ");
-                    foreach (var tag in metadata?.UniqueTags ?? new ConcurrentDictionary<string, byte>())
+                    foreach (var tag in metadata?.UniqueTags.Keys ?? new ConcurrentDictionary<string, byte>().Keys)
```
Comment (gfs) [inline src/oss-characteristics/CharacteristicTool.cs:196]:
```suggestion
                    foreach (var tag in metadata?.UniqueTags?.Keys ?? Array.Empty<string>())
```

Comment (gfs) [reply src/oss-characteristics/CharacteristicTool.cs:196]: Recommend a similar change on line 165 for the sarif result output where we are writing out the byte value.

**Comment by gfs** [comment]
@scovetta this looks stale.  Do you want to merge this?

**Comment by scovetta** [comment]
Closing, out of date.

**Pull request closed without merging**